src/model/constants.rs

```diff
@@ -53,3 +53,11 @@ pub const WEIGHT_A: f64 = 0.9;
 /// Method B: Assumes last place for unplayed games
 /// Always equals 1 - WEIGHT_A to ensure weights sum to 1
 pub const WEIGHT_B: f64 = 1.0 - WEIGHT_A;
+
+/// Constant used for applying weights to matches based on length,
+/// giving longer matches larger rating differentials.
```

Comment:
What does 'larger rating differentials' actually mean? Would be good to elaborate a touch more here.